modules/rotating-certificate-authority.adoc

```diff
@@ -6,42 +6,65 @@
 [id="rotating-certificate-authority_{context}"]
 = Rotating the etcd certificate
 
-Rotate the `etcd` certificate before it expires.
+Manually rotate the `etcd` certificate before it expires.
 
 .Procedure
 
-. Verify the remaining lifetime of the new signer certificate by running the following command:
+. Make a backup copy of the current signer certificate by running the following command:
 +
 [source,terminal]
 ----
-$ oc get secret -n openshift-etcd etcd-signer -ojsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}'
+$ oc get secret -n openshift-etcd etcd-signer -oyaml > signer_backup_secret.yaml
 ----
 
-. If the remaining lifetime is close to the current date, re-create the signer by deleting the signer and wait for the static pod roll out.
-* Delete the signer by running the following command:
+. Delete the existing signer certificate by running the following command:
 +
 [source,terminal]
 ----
 $ oc delete secret -n openshift-etcd etcd-signer
 ----
 
-* Wait for the static pod roll out by running the following command:
+. Wait for the static pod roll out by running the following command:
 +
 [source,terminal]
 ----
-$ oc wait --for=condition=Progressing=False --timeout=15m clusteroperator/etcd
+$ oc adm wait-for-stable-cluster
```

Comment:
I think you can leave the previous condition the way it was, I believe we added this through the last round of reviews